### ISSUE TITLE
In Laravel 5.5.25 does not work

### ISSUE THREAD

**Issue by fomvasss** [andersao/laravel-request-logger#26]
After command: 
```
php artisan vendor:publish --provider="Prettus\RequestLogger\Providers\LoggerServiceProvider"
```

write in console:
```
In RouteRegistrar.php line 76:
Attribute [before] does not exist.
```

I am understand that error in LoggerServiceProvider.php in method register():
```
app('router')->before(function(){
   Benchmarking::start('application');
});
```

**Comment by t1k3hu** [comment]
`composer require prettus/laravel-request-logger:dev-master`

**Comment by andersao** [comment]
I'll see that.

**Comment by andersao** [comment]
A new version was released [1.2.0](https://github.com/andersao/laravel-request-logger/releases/tag/1.2.0). Check it out.
